src/wakefs/namespace.cpp

```diff
@@ -459,4 +459,48 @@ bool setup_user_namespaces(int id_user, int id_group, bool isolate_network,
   return true;
 }
 
+int pidns_init(void *arg) {
```

Comment:
You should mark this as `[[ noreturn ]]` as well: https://en.cppreference.com/w/cpp/language/attributes/noreturn